### ISSUE TITLE
AutoDMG v1.10b587 fails to create file macOS Catalina 10.15.3 and 10.15.4

### ISSUE THREAD

**Issue by matteodurini** [MagerValp/AutoDMG#248]
## Description of issue

I'm trying to create a custom DMG adding two PKG files
- create user (gregneagle/pycreateuserpkg)
- custom one created with Packages

## Expected behavior

AutoDMG should build a DMG consisting of macOS Catalina

## Actual behavior

installer:  Error trying to locate volume at /Volumes/Preboot
Workflow failed: Build failed (Installation of 'AMPAgent.pkg' failed with return code 1. See the log for details.)

## Steps to reproduce

1. Create the user pkg file with uid=600
2. Add the Catalina app file from the dmg 
3. Add the pkg file for the user

## Log file
[AutoDMG 2020-03-26 00.20.log](https://github.com/MagerValp/AutoDMG/files/4384225/AutoDMG.2020-03-26.00.20.log)

**Comment by erikng** [comment]
Whatever AMPAgent is, that's your issue. This is unlikely an AutoDMG issue.

Thanks,
Erik Gomez
________________________________

**Comment by matteodurini** [comment]
Yes the AMPAgent entry was related to second iteration to extract the logs. I had the same problem with the pkg file generated by [pycreatpycreateuserpkge](https://github.com/gregneagle/pycreateuserpkg). Is it normal that in the log i'm getting these entries?

`2020-03-25 23:58:24 Info: installer:  The OS Install was successful.
2020-03-25 23:58:24 Info: Finding updated mount point for /dev/disk7
2020-03-25 23:58:25 Info: Install target on device /dev/disk7 is now mounted at /Volumes/Recovery
2020-03-25 23:58:25 Info: /Volumes/Macintosh HD 2
2020-03-25 23:58:25 Notice: Script phase: install 1:/Users/durini/Desktop/MAC_SILENT_INSTALL/administrator.pkg
2020-03-25 23:58:25 Debug: (IEDWorkflow:550) nextPhase, currentPhase == {
2020-03-25 23:58:25 Debug: (IEDWorkflow:550)     title = "Installing OS";
2020-03-25 23:58:25 Debug: (IEDWorkflow:550)     weight = 4294967296;
2020-03-25 23:58:25 Debug: (IEDWorkflow:550) }
2020-03-25 23:58:25 Info: Phase Installing OS with weight 4294967296 finished after 1056.755 seconds
2020-03-25 23:58:25 Notice: Starting phase: Installing administrator.pkg
2020-03-25 23:58:26 Info: Installing administrator.pkg
2020-03-25 23:58:26 Info: df: /Volumes/Recovery
2020-03-25 23:58:26 Info: /Volumes/Macintosh HD 2: No such file or directory`

it seems that mount point changed
Best regards.
Matteo

**Comment by MagerValp** [comment]
AutoDMG needs better logic to handle the mountpoint for the split system volume being changed by the OS installer. There's a workaround in pr #240 but I have a rewrite that should address this and hopefully a couple of other issues. Unfortunately it's not done yet.

**Comment by MagerValp** [comment]
However I have serious doubts that the AMPAgent installer will ever work with AutoDMG, but the pycreateuserpkg certainly should.

**Comment by matteodurini** [comment]
Hello guys, sorry for the super late reply. I did a lot of tests, always using the latest catalina version. Even using only the pkg file created by pycreateuserpkg script, AutoDMG can't complete the process. Is it related to the fact that my pkg file is not signed with a valid developer certificate? As you can see Mager in the previous log statement, the administrator.pkg was not injected. Best regards
